tests/unit/_internal/io/test_json.py

```diff
@@ -1,11 +1,21 @@
 import json
 import typing as t
+import asyncio
 from dataclasses import dataclass
 
 import numpy as np
 import pytest
 import pydantic
 
+from bentoml._internal.io_descriptors.json import JSON
+
+
+@pytest.fixture
+def loop():
```

Comment:
Since we are only using this in one function, i think a try..finally loop is sufficient enough. No need for fixture here😊

Reply:
I will address in a separate PR.